mindsdb/libs/phases/stats_generator/stats_generator.py

```diff
@@ -644,8 +666,11 @@ def run(self, input_data, hmd=None, print_logs=True):
                 stats_v2[col_name]['outliers'] = {
                     'outlier_values': stats[col_name]['lof_outliers']
                     ,'outlier_score': stats[col_name]['lof_based_outlier_score']
+                    ,'description': 'TBD'
                 }
 
+            stats_v2[col_name]['nr_warnings'] = sum([1 for x in stats_v2[col_name].values() if isinstance(x, dict) and 'warning' in x and x['warning'] is not None])
```

Comment:
Can just use a list comprehension with `len` instead of sum, but it's a matter of preference